src/scripts/videoosd.js

```diff
@@ -1,732 +1,1336 @@
-            var menuItems = streams.map(function(stream) {
-                    var opt = {
-                        name: stream.DisplayTitle,
-                        id: stream.Index
-                    };
-                    return stream.Index === currentIndex && (opt.selected = !0), opt
-                }),
-                positionTo = this;
-            require(["actionsheet"], function(actionsheet) {
-                actionsheet.show({
-                    title: globalize.translate("Subtitles"),
-                    items: menuItems,
-                    positionTo: positionTo
-                }).then(function(id) {
-                    var index = parseInt(id);
-                    index !== currentIndex && playbackManager.setSubtitleStreamIndex(index, player)
-                })
-            })
+          }
         }
+      });
+    }
 
-        function onWindowKeyDown(e) {
-            if (!currentVisibleMenu && (32 === e.keyCode || 13 === e.keyCode)) return playbackManager.playPause(currentPlayer), void showOsd();
-            switch (e.key) {
-                case "f":
-                    e.ctrlKey || playbackManager.toggleFullscreen(currentPlayer);
-                    break;
-                case "m":
-                    playbackManager.toggleMute(currentPlayer);
-                    break;
-                case "ArrowLeft":
-                case "Left":
-                case "NavigationLeft":
-                case "GamepadDPadLeft":
-                case "GamepadLeftThumbstickLeft":
-                    e.shiftKey && playbackManager.rewind(currentPlayer);
-                    break;
-                case "ArrowRight":
-                case "Right":
-                case "NavigationRight":
-                case "GamepadDPadRight":
-                case "GamepadLeftThumbstickRight":
-                    e.shiftKey && playbackManager.fastForward(currentPlayer)
-            }
-        }
+    function destroyStats() {
+      if (statsOverlay) {
+        statsOverlay.destroy();
+        statsOverlay = null;
+      }
+    }
 
-        function getImgUrl(item, chapter, index, maxWidth, apiClient) {
-            return chapter.ImageTag ? apiClient.getScaledImageUrl(item.Id, {
-                maxWidth: maxWidth,
-                tag: chapter.ImageTag,
-                type: "Chapter",
-                index: index
-            }) : null
+    function showAudioTrackSelection() {
+      var player = currentPlayer;
+      var audioTracks = playbackManager.audioTracks(player);
+      var currentIndex = playbackManager.getAudioStreamIndex(player);
+      var menuItems = audioTracks.map(function (stream) {
+        var opt = {
+          name: stream.DisplayTitle,
+          id: stream.Index
+        };
+
+        if (stream.Index === currentIndex) {
+          opt.selected = true;
         }
 
-        function getChapterBubbleHtml(apiClient, item, chapters, positionTicks) {
-            for (var chapter, index = -1, i = 0, length =
```

Comment:
```suggestion
      showOsd();
      if (enableProgressByTimeOfDay) {
```

Reply:
Again, no idea what this suggested change is implying I do. Might need updating?

Reply:
Again github misbehaving. There's an `if` with unrelated call inside - https://github.com/jellyfin/jellyfin-web/pull/117/files#diff-6d1eacc8102c82feb3807cf7a26df345R1288

This I suggest to break up in two.